ocs_ci/ocs/platform_nodes.py

```diff
@@ -1674,10 +1674,32 @@ def stop_nodes(self, nodes):
             "Stop nodes functionality is not implemented"
         )
 
-    def start_nodes(self, nodes):
-        raise NotImplementedError(
-            "Start nodes functionality is not implemented"
-        )
+    def start_nodes(self, nodes, timeout=540, wait=True):
+        """Start Azure vm instances
+
+        Args:
+            nodes (list): The OCS objects of the nodes / Azure Vm instance
```

Comment:
this arg should be from the same type. 
It should be The OCS objects of the nodes

Reply:
incorporated changes